gulp/templates/transcripts/transcript_edit.ejs

```diff
@@ -47,11 +47,17 @@
     <% } %>
 
     <% if (transcript.url) { %>
-      <p class="original-link"><a href="<%= transcript.url %>" target="_blank">View audio in Library catalogue</a></p>
+      <p class="original-link"><a href="<%= transcript.url %>" target="_blank">
```

Comment:
I've seen that a `href` was changed to `link_to` above, could that be done in this file too?
Like so
`link_to transcript.audio_item_url_title, transcript.url, target: "_blank"`

Reply:
thanks @exadeci , This is a backbone js template, can you still add the rails type helpers on that ? (I havent done much backbone)